src/main/java/com/googleinterns/zoomtube/servlets/TranscriptServlet.java

```diff
@@ -25,28 +25,17 @@
 import com.google.appengine.api.datastore.Query.FilterOperator;
 import com.google.appengine.api.datastore.Query.FilterPredicate;
 import com.google.appengine.api.datastore.Query.SortDirection;
-import com.google.common.annotations.VisibleForTesting;
 import com.google.common.collect.ImmutableList;
 import com.google.gson.Gson;
 import com.googleinterns.zoomtube.data.TranscriptLine;
 import com.googleinterns.zoomtube.utils.LectureUtil;
 import com.googleinterns.zoomtube.utils.TranscriptLineUtil;
 import java.io.IOException;
-import java.net.URL;
-import java.util.Optional;
+
```

Comment:
delete this line

Reply:
Done, not sure why this file was in here